src/eradiate/experiments/_atmosphere.py

```diff
@@ -5,7 +5,7 @@
 
 import attrs
 
-from ._core import EarthObservationExperiment, Experiment
+from ._core import EarthObservationExperiment, Experiment, measure_inside_atmosphere, _surface_converter
```

Comment:
Following up on your comment: I'm not sure either if this should go to `_core` since it's used only by a subset of the experiments; for now maybe just put it in a new `_helpers` module.